pkg/cmd/cli/restore/create.go

```diff
@@ -171,9 +171,15 @@ func (o *CreateOptions) Validate(c *cobra.Command, args []string, f client.Facto
 			return err
 		}
 	case o.ScheduleName != "":
-		if _, err := o.client.VeleroV1().Schedules(f.Namespace()).Get(o.ScheduleName, metav1.GetOptions{}); err != nil {
+		selector := fmt.Sprintf("velero.io/schedule-name=%s", o.ScheduleName)
+		scheduleItems, err := o.client.VeleroV1().Backups(f.Namespace()).List(metav1.ListOptions{LabelSelector: selector})
+		if err != nil {
 			return err
 		}
+		if len(scheduleItems.Items) == 0 {
+			fmt.Println("No backups found")
```

Comment:
I'd drop the `fmt.Println(...)` since the returned error will be bubbled up and output to stderr.

Reply:
my bad :(